lib/log-group.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable no-console */
+
+/**
+ * A simple helper to manage the print of a set of logs
+ */
+class LogGroup {
+  /**
+   * @param {object} input
+   * @param {string} input.title
+   * @param {boolean} input.isPrimary
+   */
+  constructor({title, isPrimary}) {
+    this._isPrimary = isPrimary || false;
+    this._groupTitle = title || '';
+    this._logs = [];
+    this._childGroups = [];
+  }
+
+  /**
+   *@param {object} logDetails
+   */
+  addLog(logDetails) {
+    this._logs.push(logDetails);
+  }
+
+  /**
+   * @param {object} group
+   */
+  addChildGroup(group) {
+    if (group._logs.length === 0) {
+      return;
+    }
+
+    this._childGroups.push(group);
+  }
+
+  /**
+   * prints out this log group to the console.
+   */
+  print() {
+    let isFirefox = false;
```

Comment:
How about storing this in a class variable and reusing it, since it will never change?